CHANGELOG.md

```diff
@@ -72,6 +72,11 @@ by `insecure-drop`, which still creates temporary users but does not create a
 home directory. Users who need home directory creation should either wrap `useradd`/`userdel`
 or use PAM.
 
+#### Remove restricted sessions for SSH
+
+The restricted session feature for SSH has been deprecated since Teleport 14 and 
+has been removed in Teleport 15. There is no replacement for this feature.
```

Comment:
Instead of saying "there is no replacement" I think we should say something like:

"We recommend implementing network restrictions outside of Teleport (iptables, security groups, etc)"